src/gtk-4.0/_typography.scss

```diff
@@ -155,3 +155,8 @@ label:disabled {
 .monospace {
     font-family: monospace;
 }
+
+.subtitle {
```

Comment:
Careful with this one, it is different between header bars and rows, same as with .title.

Reply:
Good call!